src/domain/app/forms/constants.ts

```diff
@@ -14,7 +14,7 @@ export enum VALIDATION_MESSAGE_KEYS {
   DATE_MAX = 'form:validation.date.max',
   TIME = 'form:validation.string.time',
   TIME_MIN = 'form:validation.time.min',
-  TIME_MAX = 'form:validation.time.min',
+  TIME_MAX = 'form:validation.time.max',
```

Comment:
This seems to have been a bug, and linting brought it up because duplicated values in the enum.